lib/salestation/web/statsd_middleware.rb

```diff
@@ -23,11 +24,13 @@ def call(env)
             'unknown-route'
           end
 
-        @statsd.timing(@metric, (Time.now - start) * 1000, tags: [
+        tags = [
           "path:#{ path }",
           "method:#{ method }",
           "status:#{ status }"
-        ])
+        ] + env.fetch(EXTRA_TAGS_ENV_KEY, [])
+
+        @statsd.timing(@metric, (Time.now - start) * 1000, tags: tags)
```

Comment:
Hmm, I though I fixed how the request duration is measured.

Reply:
Ah, that was here https://github.com/salemove/salestation/blob/master/lib/salestation/web/request_logger.rb

I know it's out of scope, but perhaps you could fix it here too?

Reply:
Can you elaborate (maybe that explains some of the bad latency metrics?)

Reply:
See this commit: https://github.com/salemove/salestation/commit/e9fef41f7447edda566b715ff03c46eced7a321d

Reply:
https://github.com/salemove/salestation/pull/39/commits/8510b63f37639c7bb0c2795cb911378416cf16f2